PairedEndSingleSampleWf.wdl

```diff
@@ -46,7 +47,7 @@ workflow PairedEndSingleSampleWorkflow {
   File wgs_calling_interval_list
   Int haplotype_scatter_count
   Int break_bands_at_multiples_of
-  Int? read_length
+  Int read_length = 250
```

Comment:
Why did you make this not optional?

Reply:
I was updating the wdl to match the most latest version here. 
https://github.com/broadinstitute/dsde-pipelines/pull/456

Reply:
@yfarjoun is it useful to adjust the `READ_LENGTH` parameter for Collect*WgsMetrics?

Reply:
Let me make my point more clear (and less snarky-seeming):

the optimal invocation would have matching actual read-length and READ_LENGTH parameter. If your parameter is too big, you'll use more memory than you need, if the parameter is too small, the program will fail and complain.

Reply:
Ok sounds like it's better to have an adjustable readlength. I’ll postpone the update of the PairedEndSingleSample, this will avoid different version of the same wdl being present in gatk-workflows and dsde-pipelines. And I will share the comments above on the dsde-pipeline pull-request.